django/thunderstore/repository/models/team.py

```diff
@@ -87,33 +87,36 @@ def strip_unsupported_characters(val: str) -> str:
     return result
 
 
-class UploaderIdentity(models.Model):
-    objects: "Manager[UploaderIdentity]"
-    members: "UploaderIdentityMemberManager[UploaderIdentityMember]"
-    owned_packages: "Manager[Package]"
+class Team(models.Model):
+    objects: "Manager[Team]"
+    members: "TeamMemberManager[TeamMember]"
 
     name = models.CharField(
         max_length=64,
         unique=True,
         validators=[PackageReferenceComponentValidator("Author name")],
     )
     is_active = models.BooleanField(default=True)
+    namespaces = models.ManyToManyField(
```

Comment:
`ManyToMany` doesn't really make sense, only a single team can own a namespace at any given time. Use a `ForeignKey` from `Namespace` to `Team` instead as a required field.